### PR TITLE
chore(issue-templates): fix yml indentation issues

### PR DESCRIPTION
**Pull request by markcaron**
## What I did

1. Fixed indentation issues in `Feature request` and `Docs edit or update request` templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/3-feature-request.yml [hunk 1]
```diff
@@ -6,26 +6,25 @@ labels:
...
+  - type: textarea
+    id: solution
+    attributes:
+      label: Suggested solution
+      description: >-
+        Describe the solution you'd like: a clear and concise description of what you want to happen.
+    validations:
+      required: true
 
   - type: textarea
```
Comment (zeroedin) [inline .github/ISSUE_TEMPLATE/3-feature-request.yml:29]: Looks like screenshot textarea is tabbed in 1 too many

File: .github/ISSUE_TEMPLATE/4-docs-request.yml [hunk 2]
```diff
@@ -41,8 +41,8 @@ body:
       required: false
 
   - type: textarea
-  id: additional-info
-  attributes:
-    label: Additional context
-    description: >-
-      Add any other context or screenshots about the request here.
\ No newline at end of file
+    id: additional-info
```
Comment (zeroedin) [inline .github/ISSUE_TEMPLATE/4-docs-request.yml:44]: this section still looks off by 2 spaces

File: .github/ISSUE_TEMPLATE/3-feature-request.yml (hunk 1 above)
Comment (markcaron) [reply .github/ISSUE_TEMPLATE/3-feature-request.yml:29]: I don't see this indentation issue in VS Code:

image

**Review by zeroedin — APPROVED** [review]
Lunar gravity, tides movement

**Pull request merged**